packages/grant-explorer/src/features/discovery/LandingPage.tsx

```diff
@@ -1,165 +1,101 @@
-
-export function useActiveRounds() {
-  const debugModeEnabled = useDebugMode();
-  return useSWR(`activeRounds`, () => getActiveRounds(debugModeEnabled));
-}
-
-export function useRoundsInApplicationPhase() {
-  const debugModeEnabled = useDebugMode();
-  return useSWR(`applicationRounds`, () =>
-    getRoundsInApplicationPhase(debugModeEnabled)
-  );
-}
+import { DefaultLayout } from "../common/DefaultLayout";
+import LandingHero from "./LandingHero";
+import { LandingSection, ViewAllLink } from "./LandingSection";
+import RoundCard from "./RoundCard";
+import { useLocation } from "react-router-dom";
 
 const LandingPage = () => {
+  const location = useLocation();
   useEffect(() => {
-    if (process.env.REACT_APP_ENV === "production") {
+    if (
+      process.env.REACT_APP_ENV === "production" &&
+      !location.search.includes("skip_redirect")
+    ) {
       window.location.replace("https://grants.gitcoin.co");
     }
-  }, []);
-
-  const [searchQuery, setSearchQuery] = useState("");
+  }, [location]);
 
-  const { isLoading: activeRoundsLoading, data: activeRounds } =
-    useActiveRounds();
+  usePrefetchRoundsMetadata();
 
-  const { isLoading: applyRoundsLoading, data: roundsInApplicationPhase } =
-    useRoundsInApplicationPhase();
-
-  const [type, setType] = useState<string>("round_type_all");
-  const [allActiveRounds, setAllActiveRounds] = useState<RoundOverview[]>([]);
-
-  useEffect(() => {
-    if (activeRounds) {
-      setAllActiveRounds(activeRounds);
-    }
-  }, [activeRounds]);
-
-  useEffect(() => {
-    if (type) {
-      filterGrantRoundByType();
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [type]);
-
-  const filterGrantRoundByType = () => {
-    const getGrantRoundType = (type: string) => {
-      return type === "round_type_direct"
-        ? ROUND_PAYOUT_DIRECT
-        : ROUND_PAYOUT_MERKLE;
-    };
-    if (type === "round_type_all") {
-      mutate("activeRounds", allActiveRounds, false);
-    }
-    if (type !== "round_type_all") {
-      const filterType = getGrantRoundType(type);
-      const filteredRounds = allActiveRounds.filter((round: RoundOverview) => {
-        return (
-          round.payoutStrategy &&
-          round.payoutStrategy.strategyName === filterType
-        );
-      });
-      mutate("activeRounds", filteredRounds, false);
-      if (searchQuery) setSearchQuery("");
-    }
-  };
-
-  useEffect(() => {
-    if (searchQuery) {
-      const timeOutId = setTimeout(
-        () => filterProjectsByTitle(activeRounds ?? [], searchQuery),
-        300
-      );
-      return () => clearTimeout(timeOutId);
-    } else {
-      filterGrantRoundByType();
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [searchQuery]);
+  const activeRounds = useActiveRounds();
+  const roundsTakingApplications = useRoundsTakingApplications();
+  // const roundsEndingSoon = useRoundsEndingSoon();
```

Comment:
Temporarily disabled to use activeRounds instead to more easily compare the data shown compared to the current build.